week5/5.1/I have no vinegret.py

```diff
@@ -0,0 +1,49 @@
+import time
+import random
+from datetime import datetime
+"""
+I have no vinegret!.
+By: Eliad Karni.
+
+The Project generates a date between two received dates.
+If the generated day is not a SHOPPING_DAY, it prints NO_VINEGRETE_DATE_OUTPUT message.
+
+used websites help:
+shopping day check -> https://www.delftstack.com/howto/python/python-datetime-day-of-week/
+                      https://www.geeksforgeeks.org/python-datetime-strptime-function/
+generate date -> https://stackoverflow.com/questions/553303/generate-a-random-date-between-two-other-dates
+"""
+#=============================== consts section ===============================
+TIME_SYNTAX = '%Y-%m-%d'
+SHOPPING_DAY = 0 #monday: 0, sunday: 6
+NO_VINEGRET_DATE_OUTPUT = "I have no vinegret!"
+#============================== function section ==============================
+#------------------------------------------------------------------------------
+def generate_date(start:str, end:str):
```

Comment:
when you copy an online example , dont assume the code conventions are good. The variable names here are not - replace them